code/modules/ghostroles/spawner/human/responseteams/vaurca/klax.dm

```diff
@@ -0,0 +1,35 @@
+/datum/ghostspawner/human/ert/klax
```

Comment:
Only the leadership role in a species restricted ERT may be whitelisted.
All the other roles must be accessible without a whitelist.